### PR TITLE
The getIframeByElement function works incorrectly for the nested iframes (close #368)

### LINKED ISSUE #368: The getIframeByElement function works incorrectly for the nested iframes

### PR DESCRIPTION
**Pull request by churkin**
\cc @LavrovArtem @helen-dikareva

### CODE REVIEW

File: src/client/utils/dom.js [hunk 1]
```diff
@@ -109,16 +110,9 @@ export function getChildVisibleIndex (select, child) {
...
-    var iframes         = currentWindow.document.getElementsByTagName('iframe');
+    var elWindow = el[INTERNAL_PROPS.processedContext];
 
-    for (var i = 0; i < iframes.length; i++) {
-        if (iframes[i].contentDocument === currentDocument)
-            return iframes[i];
-    }
-
-    return null;
+    return elWindow ? elWindow.frameElement : null;
```
Comment (LavrovArtem) [inline src/client/utils/dom.js]: use ```getFrameElement``` [client/utils/dom.js#L159](https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/utils/dom.js#L159)

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
@testcafe-build-bot \retest

**Comment by churkin** [comment]
FPR

**Comment by LavrovArtem** [comment]
What about [this](#discussion-diff-50523721)?

**Comment by churkin** [comment]
I've found a new bug #372, so tests are not yet run in the firefox.

**Comment by churkin** [comment]
FPR

**Comment by LavrovArtem** [comment]
lgtm

**Comment by helen-dikareva** [comment]
lgtm

**Pull request merged**